skins/oasis/css/core/PageHeader.scss

```diff
@@ -3,40 +3,44 @@
 @import 'skins/shared/mixins/flexbox';
 @import 'layout';
 
-.WikiaPageHeader {
-	margin: 0 10px 10px;
-	position: relative;
-
-	&.separator {
-		border-bottom: 1px solid $color-page-border;
-		padding: 0 0 15px;
+.page-header {
```

Comment:
I think it's all sorted between [829ea99](https://github.com/Wikia/app/commit/829ea998b9369645ba8b2bc5aa96f91c882a8e5e) and [4ff5617](https://github.com/Wikia/app/commit/4ff5617491d6aa833edc1e3601fbb97897aebcc3). A few areas I haven't managed to test due to issues with my devbox or not being familiar with how they work to test them:
* Places (no idea how to use it)
* Maps (no idea how to use it)
* VE (Can't get it working for some reason)
* Ads/Tracking JS
* Scavenger hunt
* Quiz

I also updated the selectors in the styles for responsive layout, although I'm not sure if they're still in use so didn't do any extra cleanup around that area.